README.md

```diff
@@ -285,6 +285,73 @@ that match exclusive check.
 
 ```attr_pattern```, ```include``` and ```exclude``` arguments work on ```objprint```, ```objstr``` and ```@add_objprint```.
 
+### Register Custom Type Formatter
+
+You can also customize how certain types of objects are displayed by registering a custom formatter function to transform an object of a specific type into a string. For example, you can print all integers in hexadecimal format by registering the ```hex()``` function for the ```int``` data type, or registering a custom ```lambda``` function. You can always unregister the formatter function if you no longer want to use it with ```unregister_formatter()```, it supports unregistering multiple data types at the same time. And if no input argument is provided to the function, it will clear all the registered functions.
```

Comment:
Please break the lines :) You can have about 100 characters per line(does not have to be precise), but don't write a super long line.